kakarot_scripts/deployment/evm_deployments.py

```diff
@@ -0,0 +1,75 @@
+import logging
+
+from kakarot_scripts.constants import EVM_ADDRESS, NETWORK, RPC_CLIENT, NetworkType
+from kakarot_scripts.utils.kakarot import deploy as deploy_evm
+from kakarot_scripts.utils.kakarot import deploy_and_fund_evm_address
+from kakarot_scripts.utils.kakarot import dump_deployments as dump_evm_deployments
+from kakarot_scripts.utils.kakarot import get_deployments as get_evm_deployments
+from kakarot_scripts.utils.starknet import call, invoke
+
+logger = logging.getLogger(__name__)
+
+
+async def deploy_evm_contracts():
+    if not EVM_ADDRESS:
+        logger.info("ℹ️  No EVM address provided, skipping EVM deployments")
+        return
```

Comment:
move into `__main__` as you did for kakaort

Reply:
dont we want this to be logged also when called from the main script?